class DivideByZero(Exception):
    print("what")


class RandomError(Exception):
    pass


def div(x, i):
    if i == 0:
        raise DivideByZero


for i in range(10):
    try:
        div(8, i)
    except RandomError:
        print("hello")

    except DivideByZero:
        print("hello")

    print(i)
